src/Al.cpp

```diff
@@ -141,16 +143,21 @@ void Initialize(int& argc, char**& argv, MPI_Comm world_comm) {
   internal::ht::init(argc, argv);
 #endif
 
-  // Add signal handlers.
-  const std::vector<int> handled_signals = {SIGILL, SIGABRT, SIGFPE,
-                                            SIGBUS, SIGSEGV};
-  static struct sigaction sa;
-  sa.sa_handler = &handle_signal;
-  sa.sa_flags = SA_RESTART;
-  sigfillset(&sa.sa_mask);
-  for (const auto& sig : handled_signals) {
-    sigaction(sig, &sa, nullptr);
+#ifdef AL_SIGNAL_HANDLER
+  // Set AL_DISABLE_SIGNAL_HANDLER to disable it.
+  if (std::getenv("AL_DISABLE_SIGNAL_HANDLER") == nullptr) {
```

Comment:
I feel inclined to point out that `AL_DISABLE_SIGNAL_HANDLER=0` would disable the signal handler.